.github/workflows/build.yml

```diff
@@ -0,0 +1,90 @@
+name: build
+on:
+  push:
+    branches:
+    - master
+    - v*
+    tags:
+    - v*
+  pull_request:
+jobs:
+  build:
+    name: Build Binaries
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Codes
+      uses: actions/checkout@v4
+
+    # Build binaries
+    - name: Run CI
+      run: make ci
+
+    # Run e2e test
+    #- name: Run E2e Test
+    #  run: make e2e-test
```

Comment:
We can add `make e2e-test` now. I will fix the flaky issue later.

Reply:
Sure. Added.